pkgs/by-name/ot/otterwiki/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  nixosTests,
+  python3,
+  fetchPypi,
+}:
+
+let
+  python = python3;
+
+in
+python.pkgs.buildPythonApplication rec {
+  pname = "otterwiki";
+  version = "2.14.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "redimp";
+    repo = "otterwiki";
+    tag = "v${version}";
+    hash = "sha256-HK9Ep8PYsDcHkmHLcKulpm/1UPpJdzELRZrkVSA4KUg=";
+  };
+
+  build-system = with python.pkgs; [ setuptools ];
+
+  pythonRelaxDeps = true;
+  dependencies = with python.pkgs; [
+    werkzeug
+    flask-login
+    flask-mail
+    sqlalchemy
+    flask-sqlalchemy
+    flask
+    jinja2
+    gitpython
+    cython
+
+    (mistune.overridePythonAttrs (oldAttrs: rec {
```

Comment:
Could you add a comment about the reason behind the override, so it's easier to know when we can drop it?